_posts/2024-02-22-Bounty-Be-a-guest-on-relevant-podcasts-and-introduce-the-audience-to-bisq.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: "$250 Bonuty - Be a guest on relevant podcasts and introduce the audience to bisq"
+author: solomon1923
+excerpt: "Bisq v1.5.0 was released earlier today, bringing SegWit support across the trading protocol as well as a slew of updates to make trading better and more convenient. <br><br>"
```

Comment:
This excerpt was copied and pasted from the last blog post that went up in Nov 2020.